docs/cloud/queries.md

```diff
@@ -0,0 +1,140 @@
+---
+title:  Queries
+sidebar_label: Queries
+---
+
+# Running Queries
+
+Once you've added a connection you will be able to run [SQL queries](sql/steampipe-sql) to explore your data, either interactively in the console, or via any PostgreSQL-compatible client - see our [integrations pages](cloud/integrations/overview) for some examples.
+
+## Exploring Schemas
+
+If you navigate to your workspace, then the **Query** tab, you'll see the interactive query console.
+
+<img src="/images/docs/cloud/cloud-query-editor.png" width="400pt"/>
+<br />
+
+From here you can either explore your schemas on the left, or dive right in and test out your own queries in the editor. The schema list supports flexible searching across all the tables in your schemas. For example, if you search for `hack new`, 
+that will find a match for the `hackernews_show_hn` table in our `hackernews` schema.
+
+<img src="/images/docs/cloud/cloud-query-schema-search.png" width="200pt"/>
+<br />
+
+Clicking the search result will automatically generate a query to select all the columns from that schema table, limited to 100 rows and will automatically run it for you.
+
+<img src="/images/docs/cloud/cloud-query-table-results.png" width="400pt"/>
+<br />
+
+If you click the `Edit` button you can amend the query, perhaps by selecting just the columns you're interested in, or adding a `where` clause to filter the results. Please note that we limit queries to 5,000 rows in the interactive query console.  
```

Comment:
is it 5000 or 3000?   I expected it to be the query API limitation, which the docs say is 3000 https://redocly.github.io/redoc/?url=https://cloud.steampipe.io/api/latest/docs/openapi.json&nocors#tag/OrgWorkspaces/operation/orgWorkspace_postQuery.  Are the api docs wrong?

Reply:
Docs are wrong. It’s 5000. We’ll  update them 👍